kornia/losses/focal.py

```diff
@@ -20,6 +20,7 @@ def focal_loss(
     gamma: float = 2.0,
     reduction: str = "none",
     weight: Optional[Tensor] = None,
+    ignore_index: int = -100,
```

Comment:
can this be set as None too ?

Reply:
I initially followed value and typing from pytorch, e.g.
https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html

considering that the kornia losses are pure python, I added a short circuit for None that saves some computation. I left the default value the same as in pytorch though. It is not a breaking change as -100 would crash if fed into one_hot. As such no existing code should have it as a valid label.